apps/dashboard/app/(app)/apis/[apiId]/keys/[keyAuthId]/new/client.tsx

```diff
@@ -2,6 +2,7 @@
 import { revalidate } from "@/app/actions";
 import { CopyButton } from "@/components/dashboard/copy-button";
 import { Loading } from "@/components/dashboard/loading";
+import { PageHeader } from "@/components/dashboard/page-header";
```

Comment:
is this used?

Reply:
Yes, it is unused. I was planning to use it. But ended up using the `Code` component instead. Removed it